pages/linux/etckeeper.md

```diff
@@ -0,0 +1,24 @@
+# etckeeper
+
+> Track system configuration files in git.
+> More information: http://etckeeper.branchable.com/.
+
+- Initialize: Run once from /etc to initialize a new repo, and add package manager hooks for automatic commits when installing and removing packages:
+
+`sudo etckeeper init`
+
+- Commit all changes in /etc:
+
+`sudo etckeeper commit {{message}}`
+
+- Run arbitrary git commands:
+
+`sudo etckeeper vcs {{command}}`
+
+- Check if there are uncommitted changes:
+
+`if (sudo etckeeper unclean); then echo dirty; else echo clean; fi`
```

Comment:
I don't know if it's a good practice to keep this function here. Other commands seem good to me.

Reply:
Ok. I added it because  `etckeeper unclean` just returns a boolean, but you can do the same thing with `sudo etckeeper git status`. Should I change it to that or remove it entirely?

Reply:
Just keep `etckeeper unclean` and explain what it does. It is upto the user to build commands on top of it.

Reply:
Yeah. Different shells will support different if statement syntax, so it's probably best to keep just the command itself and explain that the exit code is either 0 or 1 depending on the result.